src/main/plugin/iso19139.ca.HNAP/layout/layout-custom-fields.xsl

```diff
@@ -84,7 +87,7 @@
   </xsl:template>
 
   <!-- Readonly elements -->
-  <xsl:template mode="mode-iso19139" priority="2005" match="gmd:fileIdentifier|gmd:dateStamp">
+  <xsl:template mode="mode-iso19139" priority="2005" match="gmd:fileIdentifier[$schema = 'iso19139.ca.HNAP']|gmd:dateStamp[$schema = 'iso19139.ca.HNAP']">
```

Comment:
This template can be removed, as can rely in the `iso19139`  template to handle these elements. Tested working fine.

Reply:
removed